stl/inc/xloctime

```diff
@@ -651,6 +651,17 @@ protected:
     __CLR_OR_THIS_CALL ~time_get_byname() noexcept override {}
 };
 
+constexpr bool _Is_valid_fmt_specifier(const char _Specifier) {
+    constexpr char _Valid_specifiers[] = "aAbBcCdDeFgGhHIjmMnprRStTuUVwWxXyYzZ%";
+    for (const char _Valid_specifier : _Valid_specifiers) {
+        if (_Valid_specifier == _Specifier) {
+            return true;
+        }
+    }
+
+    return false;
+}

```

Comment:
I was thinking about `std::binary_search` here, but I guess this would require movement to a .cpp file which as of now does not exist for `xloctime`. Also not sure this is the correct place for this function in general. I'm accepting any guidelines or suggestions.

P.S. Also should I have __CLRCALL_OR_CDECL here?